front/src/pages/people/People.tsx

```diff
@@ -84,7 +84,7 @@ export function People() {
               data={people}
               columns={peopleColumns}
               viewName="All People"
-              viewIcon={<IconList />}
+              viewIcon={<IconList size={16} />}
```

Comment:
@charlesBochet Instead of putting `16` each time, if it's something constant across component, don't you think it's better to put it in the theme ?

Reply:
yes, we can!